karma.conf.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
     browserify: {
       watch: true,
       debug: true,
-      transform: ['babelify']
+      transform: ['es6ify']
```

Comment:
My unit tests stopped / never working with babelify and I haven't had a chance to look at it yet.